script.js

```diff
@@ -1 +1,9 @@
 "use strict";
+document.getElementById('header_burger').onclick = function() {
```

Comment:
Add event listeners instead of overwriting onclick functionality. This will overwrite any other functionality that is attached to this element.

Reply:
> Add event listeners instead of overwriting onclick functionality. This will overwrite any other functionality that is attached to this element.

Added event listeners